.github/workflows/ci.yml

```diff
@@ -16,6 +16,7 @@ jobs:
         with:
           ref: ${{ github.head_ref }}
       - name: Set up Conda env
-        uses: mamba-org/provision-with-micromamba@268cc19728703111bda67eb6e7328258ff6eb8b6
+        uses: mamba-org/setup-micromamba@63849fa5f38bd7edfb2c4b518206ea9eee3502d1
         with:
+          environment-file: environment.yml
           environment-name: test-env
```

Comment:
Just out of curiosity, since now an explicit file is required is the `environment-name` also strictly required?

Reply:
No, it's not strictly required. 
If `environment-name` is specified, the one specified in `environment.yml` will be overwritten. In this case, there is no name in `environment.yml` so this would fail if we didn't specify `environment-name`.

Reply:
It is strictly required if you create an environment without providing `environment.yml`, see [here](https://github.com/mamba-org/setup-micromamba#create-environment-from-environment-specs).